README.md

```diff
@@ -1 +1,204 @@
-# OpenSLO
\ No newline at end of file
+# OpenSLO
+
+## Table of Contents
+
+- [OpenSLO](#openslo)
+  - [Introduction](#introduction)
+  - [Specification](#specification)
+    - [Goals](#goals)
+    - [Object Types](#object-types)
+      - [General Schema](#general-schema)
+        - [Notes](#notes)
+      - [SLO](#slo)
+        - [Notes](#notes-1)
+        - [Objectives](#objectives)
+      - [Service](#service)
+
+## Introduction
+
+The intent of this document is to outline the OpenSLO specification.
+
+The goal of this project is to provide an open specification for defining and
+interfacing with SLOs to allow for a common, giving a set vendor-agnostic
+solution to defining and tracking SLOs. Platform specific implimentation details
```

Comment:
s/implimentation/implementation/

As a matter of interest, what _could_ be platform specific about this? Are you talking about the potential interactions with the monitoring system?

Reply:
> As a matter of interest, what could be platform specific about this? Are you talking about the potential interactions with the monitoring system?

Yeah, exactly those sorts of things.  Like for us, we have agent configuration, and that might look different depending on the vendor

Reply:
fixed `implementation`